apps/web/components/molecules/modal/loginModal.tsx

```diff
@@ -0,0 +1,27 @@
+import { Modal } from 'molecules/modal/modal';
```

Comment:
loginModal wykorzystuję molecule w takim wypadku najlepiej jest to wrzucić do organizmu